tests/keyvalue/test.bats

```diff
@@ -0,0 +1,70 @@
+# PURPOSE:
+# I will test the following commands: kv get/set, clean, del, expire/ttl, and list in the first set of tests.
+# I will later add tests for getMany, llen, lrange, rpush, and setMany.
+load ../test_setup.bash
+
+# constants?
+KEY_NAME="SOMEKEY" # random name?
+
+setup_file() { # kv delete being tested here
+	$NIM key-value delete $KEY_NAME
+}
+
+teardown_file() { # kv delete also being tested here
+	$NIM key-value delete $KEY_NAME
+}
+
+@test "kv set/get" { # kv set/get test
+	testValue=$(date)
```

Comment:
```suggestion
  testValue=$(date)
```

Replace tabs with white space.
@jthomas I guess we don't run scan code on these files - is there some convenient way to enforce formatting for bats?

Reply:
@rabbah Some Googling leads me to.. https://github.com/mvdan/sh I wonder if this would work?